src/styles/reset.scss

```diff
@@ -0,0 +1,9 @@
+html {
+  box-sizing: border-box;
+  font-family: "Roboto", sans-serif;
+  color: $blue-dark-color;
```

Comment:
It looks strange, that you add this color to html. Mostly this color is used as decorative.